shop.py

```diff
@@ -0,0 +1,75 @@
+splash = '''
+
+                    ███████╗██╗  ██╗ ██████╗ ██████╗     ██████╗  ██████╗ ████████╗
+                    ██╔════╝██║  ██║██╔═══██╗██╔══██╗    ██╔══██╗██╔═══██╗╚══██╔══╝
+                    ███████╗███████║██║   ██║██████╔╝    ██████╔╝██║   ██║   ██║
+                    ╚════██║██╔══██║██║   ██║██╔═══╝     ██╔══██╗██║   ██║   ██║
+                    ███████║██║  ██║╚██████╔╝██║         ██████╔╝╚██████╔╝   ██║
+                    ╚══════╝╚═╝  ╚═╝ ╚═════╝ ╚═╝         ╚═════╝  ╚═════╝    ╚═╝
+
+                       - = https://github.com/werewolves-devs/werewolf_bot = -
+
+'''
+
+splashes = [
+'It\'s only selling your soul to the devil!',
+'They\'re cheap, honest!'
+]
+
+import discord
+import asyncio
+import random
+import json
+from emoji import emojize, demojize
+
+# Import config data
+import config
+
+client = discord.Client()
+
+async def setup_shop(shop_config):
+    # Sends the shop message and adds reactions
+    embed = discord.Embed(title="Shop (Page 1/1)", description=shop_config["shop_description"], color=0x00ff00)
+    for item in shop_config["items"]:
```

Comment:
Can you make this into a Mailbox feature, please?

Reply:
can I make what into a mailbox feature?

Reply:
The `setup_shop` function. Also, couldya give a default value to `shop_config` for convenience?

Reply:
`shop_config` is set - its values come from the file `shop.json`

Reply:
Also - Why would we want to make `setup_shop` a Mailbox() - I can't think of a situation where one would want to instantiate another shop - which would invalidate all others

Reply:
Well, that would mean all purchases would need to be made in public.

Reply:
Yes - I thought that was intended